Sources/HummingbirdLambda/CollateResponseBodyWriter.swift

```diff
@@ -0,0 +1,26 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2023 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import HummingbirdCore
+
+internal actor CollateResponseBodyWriter: HBResponseBodyWriter {
```

Comment:
`HBResponseBodyWriter` does not conform to Sendable so this doesn't need to be an actor.